packages/tools/src/tools/annotation/PlanarFreehandROITool.ts

```diff
@@ -743,11 +744,16 @@ class PlanarFreehandROITool extends AnnotationTool {
         continue;
       }
 
-      const { imageData, metadata } = image;
-      const canvasCoordinates = points.map((p) => viewport.worldToCanvas(p));
+      const { imageData, metadata, hasPixelSpacing } = image;
+      const imageId = viewport.getCurrentImageId();
+      const imageCoordinates = points.map((p) => csUtils.worldToImageCoords(imageId, p));
```

Comment:
is points in world? why do we need to go to imageCoords?

Reply:
The polyline `calculateAreaOfPoints` method only accepts point2 as input